config/initializers/sidekiq.rb

```diff
@@ -1,11 +1,9 @@
-if Rails.env.production? || Rails.env.staging?
-  redis_jobs_url = ENV.fetch("OFN_REDIS_JOBS_URL", "redis://localhost:6381/0")
+redis_jobs_url = ENV.fetch("OFN_REDIS_JOBS_URL", "redis://localhost:6381/0")
```

Comment:
I guess we'll have to think a bit about how this works in dev. We currently don't require devs to install Redis locally (most of use don't use the Docker setup) so we'll have to update the docs and think about recommendations for setting `OFN_REDIS_JOBS_URL` if people want to test mails in dev?

And I guess we don't want this enabled in the test environment? Or maybe it doesn't matter, as we're not running the service anyway, and the test jobs are al going through an `ActiveJob` wrapper that's test-env-safe?

:thinking:

Reply:
> We currently don't require devs to install Redis locally

It's not like its mandatory but you'll need it if you want to run background jobs. In all projects I worked on it was assumed one day or the other you would need to boot it. Personally, I think that's where Docker (and docker-compose) shines because it's just a command away.

> And I guess we don't want this enabled in the test environment? 

that's a totally separate environment with its separate config, where jobs are processed inline (I believe). See https://edgeapi.rubyonrails.org/classes/ActiveJob/QueueAdapters/TestAdapter.html.